jinx.el

```diff
@@ -162,6 +162,11 @@ checking."
   "List of modes included by `global-jinx-mode'."
   :type '(repeat symbol))
 
+(defcustom jinx-max-distance-for-suggestion
```

Comment:
I know the name of this variable is not as satisfactory. I'm expecting to change it if you could suggest a better one.